src/components/look/look-list/style-tag-list/index.tsx

```diff
@@ -0,0 +1,66 @@
+import styled from 'styled-components';
+import { useState } from 'react';
+import { message } from 'antd';
+
+import { Button } from '@src/component/atoms';
+
+import { StyleTag } from '@src/types';
+import styleTagsData from '@src/data/style-tags.json';
+import { LookService } from '@src/services';
```

Comment:
```suggestion
import { StyleTag } from '@src/types';
import { LookService } from '@src/services';

import STYLE_TAGS from '@src/data/style-tags.json';
```

상수 변수는 대문자 스네이크 케이스로 작성해주세요!
import문은 분리해서 가장 아래에 작성해주세요

Reply:
f512ca2